inventory/sample/group_vars/k8s-cluster.yml

```diff
@@ -112,7 +112,7 @@ kube_apiserver_insecure_port: 8080 # (http)
 
 # Kube-proxy proxyMode configuration.
 # Can be ipvs, iptables
-kube_proxy_mode: ipvs
+kube_proxy_mode: iptables
```

Comment:
seems odd when you are removing istio.

Reply:
yes, test install failed with ipvs.

Reply:
We didn't run in any issues yet running `kube-proxy` using `ipvs` with Kubespray `master` (so, Kubernetes 1.10.something), so this seems fishy...